websites/demo-image/src/main.ts

```diff
@@ -1,20 +1,9 @@
 import { checkWebPSupport, load } from "@fepack/image";
 
-const images = [
-  { defaultSrc: "/options_resize.png" },
-  {
-    webpSrc: "/information_resize.webp",
-    defaultSrc: "/information_resize.png",
-  },
-];
-
-(async function () {
-  const webPSupport = await checkWebPSupport();
-
-  console.log(webPSupport);
-})();
-
-load(images);
+checkWebPSupport().then(console.log);
+load("/options_resize.png");
+load("/information_resize.webp");
+load("/information_resize.png");
```

Comment:
> Since load has changed, demo-image also needs to be updated. Currently, there's an error when building.
> 
> demo-image change plz <img alt="스크린샷 2023-09-25 오전 3 39 47" width="790" src="https://user-images.githubusercontent.com/77133565/270184928-f8208d66-82f3-4481-b9ce-891a583d9f8b.png">

I changed like this